airbyte-webapp/package.json

```diff
@@ -8,13 +8,13 @@
   },
   "scripts": {
     "prepare": "cd .. && husky install airbyte-webapp/.husky",
-    "prestart": "pnpm run generate-client",
+    "prestart": "TS_NODE_TRANSPILE_ONLY=true pnpm run generate-client",
     "start": "NODE_OPTIONS='-r ./scripts/dev-overwrites.js' vite",
-    "prestart:cloud": "pnpm run generate-client",
+    "prestart:cloud": "TS_NODE_TRANSPILE_ONLY=true pnpm run generate-client",
     "start:cloud": "AB_ENV=${AB_ENV-frontend-dev} NODE_OPTIONS='-r ./scripts/environment.js -r ./scripts/dev-overwrites.js' vite",
     "prebuild": "pnpm run generate-client",
```

Comment:
`prebuild` runs the generation script with type checking; the other, more development-focused commands which run it (like `prestart`) skip type checking with `TS_NODE_TRANSPILE_ONLY=true` to optimize for execution speed.